client/app/components/blueprint-details-modal/order-list.directive.js

```diff
@@ -38,20 +38,18 @@
         list: '='
       },
       templateUrl: 'app/components/blueprint-details-modal/order-list.directive.html',
-      controller: ['$scope', '$filter', OrderListController],
-      controllerAs: 'vm'
+      controller: OrderListController
     };
 
     return directive;
   }
 
   function OrderListController($scope, $filter) {
-    var vm = this;
-    vm.listDisabled = false;
+    $scope.listDisabled = false;
```

Comment:
@AllenBW only changed the javascript and not the corresponding HTML, and none of the blueprint functionality was tested before the PR was merged.   Long term I think one blueprint component at a time should be converted and tested.  @serenamarie125 should I create corresponding JIRA stories for converting components to pass eslint?  -thanks